nextflow_schema.json

```diff
@@ -296,6 +296,146 @@
                     "fa_icon": "fas fa-bacon"
                 }
             }
+        },
+        "preprocessing": {
+            "title": "Preprocessing",
+            "type": "object",
+            "description": "Removal of adapters, paired-end merging, poly-G removal etc.",
+            "default": "",
+            "properties": {
+                "skip_preprocessing": {
+                    "type": "boolean",
+                    "description": "Specify to skip all preprocessing steps (adapter removal, paired-end merging, poly-g trimming etc).",
+                    "help_text": "Specify to skip all preprocessing steps (adapter removal, paired-end merging, poly-g trimming etc).\n\nThis will also mean you will only get one set of FastQC results (of the input reads).",
+                    "fa_icon": "fas fa-forward"
+                },
```

Comment:
Does this not require a default value of `false`? Is the default of the section (`""`)  enough?

Reply:
Apparently default in booelan is assumed to be false 🤷‍♀ I've just rebuilt the schema and same applies (and also checked other pipelines)